could not determine kind of name for C.frida_script_enable_debugger_sync

../gowork/pkg/mod/github.com/frida/frida-go@v0.7.2/frida/script.go:97:2: could not determine kind of name for C.frida_script_disable_debugger_sync
../gowork/pkg/mod/github.com/frida/frida-go@v0.7.2/frida/script.go:86:2: could not determine kind of name for C.frida_script_enable_debugger_sync

Just upgraded to frida 16.3.3 and now I'm seeing this issue:

```
# github.com/frida/frida-go/frida
../../../../go/pkg/mod/github.com/frida/frida-go@v0.10.1/frida/variants_to_go.go:56:13: error: incompatible integer to pointer conversion passing 'gsize' (aka 'unsigned long') to parameter of type 'gsize *' (aka 'unsigned long *') [-Wint-conversion]
                                                                           (gsize)n_elements,
                                                                           ^~~~~~~~~~~~~~~~~
/usr/local/include/frida-core.h:13833:96: note: passing argument to parameter 'n_elements' here
                                                                         gsize                *n_elements,
```

Note, I'm using the latest go-frida version: `require github.com/frida/frida-go v0.10.1`.

I've also copied the `frida-core.h` and `libfrida-core.a` files from the Frida Core Devkit 16.3.3 for MacOS arm64 to their respective directories according to the README installation instructions.

--- a/frida/variants_to_go.go
+++ b/frida/variants_to_go.go
@@ -53,7 +53,7 @@ static char* read_byte_array(GVariant *variant, int * n_elements)
 	guint8 * array = NULL;
 
 	array = g_variant_get_fixed_array (variant,
-									   (gsize)n_elements,
+									   (gsize *)&n_elements,
 									   sizeof(guint8));
 	return (char*)array;
 }

--- a/frida/variants_to_go.go
+++ b/frida/variants_to_go.go
@@ -53,7 +53,7 @@ static char* read_byte_array(GVariant *variant, int * n_elements)
 	guint8 * array = NULL;
 
 	array = g_variant_get_fixed_array (variant,
-									   (gsize *)&n_elements,
+									   (gsize *)n_elements,
 									   sizeof(guint8));
 	return (char*)array;
 }

